zwave_js_server/client.py

```diff
@@ -39,6 +41,10 @@ class InvalidState(Exception):
     """Exception raised when data gets in invalid state."""
 
 
+class InvalidServerVersion(Exception):
+    """Exception raised when connected to server with incompatible major version."""
```

Comment:
The check also takes minor version into account.
```suggestion
    """Exception raised when connected to server with incompatible version."""
```